tests/behaviour/qbwriter.feature

```diff
@@ -66,53 +66,53 @@ Feature: Test outputting CSV-Ws with Qb flavouring.
     And csvlint validation of all CSV-Ws should succeed
     And csv2rdf on all CSV-Ws should succeed
     And the RDF should contain
-    """
+      """
       @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
       @prefix qudt: <http://qudt.org/schema/qudt/>.
       @prefix om2: <http://www.ontology-of-units-of-measure.org/resource/om-2/>.
 
       <{{rdf_input_directory}}/some-qube.csv#attribute/new-attribute/pending>
-        a rdfs:Resource;
-        rdfs:label "pending"@en.
+      a rdfs:Resource;
+      rdfs:label "pending"@en.
 
       <{{rdf_input_directory}}/some-qube.csv#attribute/new-attribute/final>
-        a rdfs:Resource;
-        rdfs:label "final"@en.
+      a rdfs:Resource;
+      rdfs:label "final"@en.
 
       <{{rdf_input_directory}}/some-qube.csv#attribute/new-attribute/in-review>
-        a rdfs:Resource;
-        rdfs:label "in-review"@en.
+      a rdfs:Resource;
+      rdfs:label "in-review"@en.
 
       <{{rdf_input_directory}}/some-qube.csv#unit/some-unit>
-        a qudt:Unit, om2:Unit;
-        rdfs:label "Some Unit"@en.
-    """
+      a qudt:Unit, om2:Unit;
+      rdfs:label "Some Unit"@en.
+      """
 
   Scenario: QbCube extended units (and new base units) should be serialised correctly.
     Given a single-measure QbCube named "Some Qube" with one new unit extending another new unit
     When the cube is serialised to CSV-W
     Then csvlint validation of all CSV-Ws should succeed
     And csv2rdf on all CSV-Ws should succeed
     And the RDF should contain
-    """
+      """
       @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
       @prefix qudt: <http://qudt.org/schema/qudt/>.
       @prefix om2: <http://www.ontology-of-units-of-measure.org/resource/om-2/>.
       @prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
 
       <{{rdf_input_directory}}/some-qube.csv#unit/some-base-unit>
-        a qudt:Unit, om2:Unit;
-        rdfs:label "Some Base Unit"@en.
+      a qudt:Unit, om2:Unit;
+      rdfs:label "Some Base Unit"@en.
 
       <{{rdf_input_directory}}/some-qube.csv#unit/some-extending-unit>
-        a qudt:Unit, om2:Unit;
-        qudt:isScalingOf <{{rdf_input_directory}}/some-qube.csv#unit/some-base-unit>;
-        qudt:hasQuantityKind <http://some-quantity-kind>;
-        qudt:conversionMultiplier "25.123123"^^xsd:float;
-        om2:hasUnit <{{rdf_input_directory}}/some-qube.csv#unit/some-base-unit>;
-        om2:hasFactor "1000.0"^^xsd:float;
-        rdfs:label "Some Extending Unit"@en.
-    """
+      a qudt:Unit, om2:Unit;
```

Comment:
Can you try not to alter the indentation in future? It makes it harder to see the pattern without it.